config/locales/diaspora/ru.yml

```diff
@@ -1403,4 +1407,4 @@ ru:
   welcome: "Добро пожаловать!"
   will_paginate:
     next_label: "вперед &raquo;"
-    previous_label: "&laquo; назад"
\ No newline at end of file
+    previous_label: "&laquo; назад"
```

Comment:
Please only edit the English locale, translations are handled via WebTranslateIt.